src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -2228,6 +2228,56 @@ public void IfAndroidJarDoesNotExistThrowXA5207 ()
 			Directory.Delete (AndroidSdkDirectory, recursive: true);
 		}
 
+		[Test]
+		public void ValidateUseLatestAndroid ()
+		{
+			var path = Path.Combine ("temp", TestName);
+			var referencesPath = CreateFauxReferencesDirectory (Path.Combine (path, "xbuild-frameworks"), new ApiInfo [] {
+				new ApiInfo () { Id = 23, Level = 23, Name = "Marshmallow", FrameworkVersion = "v6.0", Stable = true },
+				new ApiInfo () { Id = 26, Level = 26, Name = "Oreo", FrameworkVersion = "v8.0", Stable = true },
+				new ApiInfo () { Id = 27, Level = 27, Name = "Oreo", FrameworkVersion = "v8.1", Stable = false },
+			});
```

Comment:
For the test case, we need to add:

```csharp
new ApiInfo () { Id = "P", Level = 28, Name = "P", FrameworkVersion="v8.99", Stable = false },
```

(This is why `AndroidApiInfo.xml` differentiates `Id` from `Level`. They're the same for all *stable* API levels, but for preview API levels *they are not*.)